lib/graph3d/Graph3d.js

```diff
@@ -1,13 +1,19 @@
-var Emitter = require('emitter-component');
-var DataSet = require('../DataSet');
-var DataView = require('../DataView');
-var util = require('../util');
-var Point3d = require('./Point3d');
-var Point2d = require('./Point2d');
-var Camera = require('./Camera');
-var Filter = require('./Filter');
-var Slider = require('./Slider');
-var StepNumber = require('./StepNumber');
+var Emitter      = require('emitter-component');
```

Comment:
I hope i don't sound too nitpicky but I've always preferred required libraries ordered alphabetically.
You don't have to rearrange it, but I do think it looks and reads better (similar to package.json in npm).